Modules/Installer/Core.lua

```diff
@@ -164,7 +164,7 @@ function IS:ElvUIProfileDialog()
   local function createNewProfile(name)
     if strtrim(name) == "" then return end
 
-    if E.data:IsDualSpecEnabled() then
+    if not TXUI.IsClassic and E.data:IsDualSpecEnabled() then
```

Comment:
Does classic not have dual spec? Was that a tbc/wrath feature?

Reply:
It was wotlk feature, if I remember correctly

Reply:
Dual spec was a Wrath feature. There was a lot of people complaining that they wanted it in classic and tbc but they didn't give it to us afaik.